<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Cliente</class>
 <widget class="QDialog" name="Cliente">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>857</width>
    <height>556</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog{
	background-color: #2A4161;
}</string>
  </property>
  <widget class="QPushButton" name="reservaciones">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>91</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color: rgba(0,0,0,0);
	border: none;
	image: url(:/img/img/calendar.png);
}

QPushButton:Hover{
	background-color: rgba(0,0,0,0);
	border: none;
	image: url(:/img/img/calendar_hover.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>150</width>
     <height>150</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="modificar">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>91</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color: rgba(0,0,0,0);
	border: none;
	image: url(:/img/img/modificar.png);
}

QPushButton:Hover{
	background-color: rgba(0,0,0,0);
	border: none;
	image: url(:/img/img/modificar_hover.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="estadoCuenta">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>350</y>
     <width>91</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color: rgba(0,0,0,0);
	border: none;
	image: url(:/img/img/estadoCuenta.png);
}

QPushButton:Hover{
	background-color: rgba(0,0,0,0);
	border: none;
	image: url(:/img/img/estadoCuenta_Hover.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>0</y>
     <width>731</width>
     <height>561</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #B8BDD9;</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>331</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(0,0,0,0);
font: 75 30pt &quot;Corbel&quot;;
color: white;</string>
     </property>
     <property name="text">
      <string>Tipo de reservaci√≥n:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>50</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: white;
border: 1px solid #A1C1E6;
font: 10pt &quot;Corbel&quot;;</string>
     </property>
     <item>
      <property name="text">
       <string>Estandar</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Mensual</string>
      </property>
     </item>
    </widget>
    <widget class="QStackedWidget" name="stackedWidget_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>100</y>
       <width>721</width>
       <height>431</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="Estandar">
      <widget class="QWidget" name="widget" native="true">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>20</y>
         <width>661</width>
         <height>421</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QWidget{
	background-color: white;
}

QLabel{
	font: 12pt &quot;Corbel&quot;;
	color: black,
}

QLineEdit{
	border: 1px solid #A1C1E6;
}

QTimeEdit{
	border: 1px solid #A1C1E6;
	font: 10pt &quot;Corbel&quot;;
}

QCalendarWidget{
	border: 1px solid #A1C1E6;
}</string>
       </property>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>320</y>
          <width>121</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Hora de llegada:</string>
        </property>
       </widget>
       <widget class="QTimeEdit" name="estLlegada">
        <property name="geometry">
         <rect>
          <x>191</x>
          <y>320</y>
          <width>118</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>350</x>
          <y>320</y>
          <width>101</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Hora de salida:</string>
        </property>
       </widget>
       <widget class="QTimeEdit" name="estSalida">
        <property name="geometry">
         <rect>
          <x>451</x>
          <y>320</y>
          <width>118</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QCalendarWidget" name="calendarioEst">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>70</y>
          <width>341</width>
          <height>231</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">border: 1px solid #A1C1E6;</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>310</x>
          <y>40</y>
          <width>47</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>Fecha:</string>
        </property>
       </widget>
       <widget class="QPushButton" name="agendarEst">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>360</y>
          <width>111</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
	background-color: #2A4161;
	color: white;
	font: 10pt &quot;Corbel&quot;;
	border: none;
}

QPushButton:Hover{
	background-color: #C2C9F2;
}</string>
        </property>
        <property name="text">
         <string>Agendar</string>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QWidget" name="Mensual">
      <widget class="QWidget" name="widget_2" native="true">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>20</y>
         <width>661</width>
         <height>421</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QWidget{
	background-color: white;
}

QLabel{
	font: 12pt &quot;Corbel&quot;;
	color: black,
}

QLineEdit{
	border: 1px solid #A1C1E6;
}

QTimeEdit{
	border: 1px solid #A1C1E6;
	font: 10pt &quot;Corbel&quot;;
}

QCalendarWidget{
	border: 1px solid #A1C1E6;
}</string>
       </property>
       <widget class="QCalendarWidget" name="mensualInicio">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>110</y>
          <width>341</width>
          <height>231</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">border: 1px solid #A1C1E6;
color: black;</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>80</y>
          <width>201</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Seleccione su fecha de inicio</string>
        </property>
       </widget>
       <widget class="QPushButton" name="agendarMensual">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>350</y>
          <width>111</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
	background-color: #2A4161;
	color: white;
	font: 10pt &quot;Corbel&quot;;
	border: none;
}

QPushButton:Hover{
	background-color: #C2C9F2;
}</string>
        </property>
        <property name="text">
         <string>Agendar</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QWidget" name="widget_5" native="true">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>681</width>
       <height>531</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget{
	background-color: white;
}

QLabel{
	font: 12pt &quot;Corbel&quot;;
	color: black,
}

QLineEdit{
	border: 1px solid #A1C1E6;
}

QTimeEdit{
	border: 1px solid #A1C1E6;
	font: 10pt &quot;Corbel&quot;;
}

QCalendarWidget{
	border: 1px solid #A1C1E6;
}</string>
     </property>
     <widget class="QTableWidget" name="Registros">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>641</width>
        <height>231</height>
       </rect>
      </property>
     </widget>
     <widget class="QTimeEdit" name="NuevaLlegada">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>380</y>
        <width>118</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>184</x>
        <y>296</y>
        <width>161</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Reservaci√≥n a modificar:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>204</x>
        <y>420</y>
        <width>141</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Nueva hora de salida:</string>
      </property>
     </widget>
     <widget class="QTimeEdit" name="NuevaSalida">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>420</y>
        <width>118</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>380</y>
        <width>161</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Nueva Hora de llegada:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>460</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color: #2A4161;
	color: white;
	font: 10pt &quot;Corbel&quot;;
	border: none;
}

QPushButton:Hover{
	background-color: #C2C9F2;
}</string>
      </property>
      <property name="text">
       <string>Aceptar</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>255</x>
        <y>340</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Nueva Fecha:</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="NuevaFecha">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>340</y>
        <width>110</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="Reservaciones">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>295</y>
        <width>181</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_3">
    <widget class="QWidget" name="widget_4" native="true">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>661</width>
       <height>481</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget{
	background-color: white;
}

QLabel{
	font: 12pt &quot;Corbel&quot;;
	color: black,
}

QLineEdit{
	border: 1px solid #A1C1E6;
}

QTimeEdit{
	border: 1px solid #A1C1E6;
	font: 10pt &quot;Corbel&quot;;
}

QCalendarWidget{
	border: 1px solid #A1C1E6;
}</string>
     </property>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>150</y>
        <width>511</width>
        <height>161</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="Actualizar">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>370</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color: #2A4161;
	color: white;
	font: 10pt &quot;Corbel&quot;;
	border: none;
}

QPushButton:Hover{
	background-color: #C2C9F2;
}</string>
      </property>
      <property name="text">
       <string>Actualizar</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>0</y>
        <width>141</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Extender reservacion</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>80</y>
        <width>211</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Edite la nueva hora de salida </string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>460</y>
     <width>75</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color: rgba(0,0,0,0);
	border: none;
	image: url(:/img/img/logout.png);
}

QPushButton:Hover{
	background-color: rgba(0,0,0,0);
	border: none;
	image: url(:/img/img/logout_hover.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>220</y>
     <width>121</width>
     <height>121</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color: rgba(0,0,0,0);
	border: none;
	image: url(:/img/img/extender.png);
}

QPushButton:Hover{
	background-color: rgba(0,0,0,0);
	border: none;
	image: url(:/img/img/extender_hover.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
